docs/source/athena-arch.rst

```diff
@@ -1,16 +1,12 @@
-Athena Deployment
-=================
+Athena Architecture
+===================
 
 Overview
 --------
 
-After the initial `Athena Setup <athena-setup.html>`_, a new Lambda function must be deployed.
+The Athena Partition Refresh function exists to periodically refresh Athena tables, enabling the searchability of data.
```

Comment:
I don't think searchability is a proper word. In any case, this sentence is kinda awkward. Maybe something along the lines of "The Athena Refresh function periodically refreshes your Athena tables, giving you the ability to run SQL-style queries over historical log and alert data"

Reply:
the internets say otherwise :) but you're right this could be improved - I'll make a change for it!

Reply:
Haha I was just judging by the spellcheck as I was typing it. But it does seem to be in many dictionaries